### ISSUE TITLE
运行程序时报错Model name '../bert_models/scibert_uncased'没有找到bert-base-uncased等

### ISSUE THREAD

**Issue by zhang-g-z** [thunlp/PL-Marker#57]
`06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   Model name '../bert_models/scibert_uncased' not found in model shortcut name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc, bert-base-german-dbmdz-cased, bert-base-german-dbmdz-uncased, bert-base-finnish-cased-v1, bert-base-finnish-uncased-v1, bert-base-dutch-cased). Assuming '../bert_models/scibert_uncased' is a path, a model identifier, or url to a directory containing tokenizer files.
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   Didn't find file ../bert_models/scibert_uncased/added_tokens.json. We won't load it.
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   Didn't find file ../bert_models/scibert_uncased/special_tokens_map.json. We won't load it.
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   Didn't find file ../bert_models/scibert_uncased/tokenizer_config.json. We won't load it.
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   loading file ../bert_models/scibert_uncased/vocab.txt
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   loading file None
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   loading file None
06/17/2023 14:41:20 - INFO - transformers.tokenization_utils -   loading file None
06/17/2023 14:41:20 - INFO - transformers.modeling_utils -   loading weights file ../bert_models/scibert_uncased/pytorch_model.bin
06/17/2023 14:41:35 - INFO - transformers.modeling_utils -   Weights of BertForACEBothOneDropoutSub not initialized from pretrained model: ['ner_classifier.weight', 'ner_classifier.bias', 're_classifier_m1.weight', 're_classifier_m1.bias', 're_classifier_m2.weight', 're_classifier_m2.bias']
06/17/2023 14:41:35 - INFO - transformers.modeling_utils -   Weights from pretrained model not used in BertForACEBothOneDropoutSub: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.predictions.decoder.bias', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']
06/17/2023 14:41:35 - INFO - __main__ -    subject_id = 2950, object_id = 2567, mask_id = 104
06/17/2023 14:41:35 - INFO - __main__ -   Training/evaluation parameters Namespace(adam_epsilon=1e-08, alpha=1.0, att_left=False, att_right=False, cache_dir='', config_name='', data_dir='scierc', dev_file='dev.json', device=device(type='cpu'), do_eval=True, do_lower_case=True, do_train=True, eval_all_checkpoints=True, eval_logsoftmax=True, eval_softmax=False, eval_unidirect=False, evaluate_during_training=True, fp16=True, fp16_opt_level='O1', gradient_accumulation_steps=1, learning_rate=2e-05, lminit=False, local_rank=-1, logging_steps=5, max_grad_norm=1.0, max_pair_length=16, max_seq_length=256, max_steps=-1, model_name_or_path='../bert_models/scibert_uncased', model_type='bertsub', n_gpu=0, no_cuda=False, no_sym=False, no_test=False, num_train_epochs=10.0, output_dir='scire_models/scire-scibert-42', overwrite_cache=False, overwrite_output_dir=True, per_gpu_eval_batch_size=16, per_gpu_train_batch_size=8, save_results=False, save_steps=2500, save_total_limit=1, seed=42, server_ip='', server_port='', shuffle=False, test_file='sciner_models/PL-Marker-scierc-scibert-42/ent_pred_test.json', tokenizer_name='', train_file='train.json', use_ner_results=True, use_typemarker=False, warmup_steps=-1, weight_decay=0.0)`
